rust-rocket/src/types.rs

```diff
@@ -117,10 +117,7 @@ impl JWTClaims {
   /// Method that checks if the JWT has expired.
   /// This is has a max age of 5 minutes.
   pub fn check_expired(self) -> Result<Self, JWTError> {
-    let exp = DateTime::<Utc>::from_utc(
```

Comment:
I'd keep the .ok_or(JWTError::Expired)? vs using expect. It is a prototype but I prefer to avoid using non total functions unless in a test function.

Reply:
Hi @darrell-roberts I have switched back to use the approach you suggested.